great_expectations/cli/toolkit.py

```diff
@@ -126,8 +126,6 @@ def get_default_expectation_suite_name(
     suite_name: str
     if data_asset_name:
         suite_name = f"{data_asset_name}.warning"
-    elif batch_request:
-        suite_name = f"batch-{BatchRequest(**batch_request).id}"
```

Comment:
The `batch_request` may not be a valid `BatchRequest` so we can't use the object's id as a default suite name.

Reply:
could we `try/except` or split out the instantiation of the object and retrieval of `id` into two separate lines?

Reply:
I don't think so, because if `data_asset_name` is missing `BatchRequest` instantiation will always fail.

Reply:
I take that back...I think it is possible that there is a mismatch between the `data_asset_name` passed to this function and the `data_asset_name` defined in the `batch_request`. I have made updates to this block. ✅